examples/node/pdf2png/pdf2png.js

```diff
@@ -50,14 +50,20 @@ NodeCanvasFactory.prototype = {
 
```

Comment:
Maybe we should also export the various canvas/CMap-factories in the API as well, such that we could remove things like e.g. https://github.com/mozilla/pdf.js/blob/9993397e30bdd6c3e63800753da3e4c971fa2f35/examples/node/pdf2png/pdf2png.js#L20-L49
However, I'm not sure if that's (generally) desirable and it also felt somewhat orthogonal to the rest of the patch...